app/javascript/packs/selectors/answersSelector.js

```diff
@@ -1,3 +1,22 @@
 export function getAnswerForQuestionAndBuilding(questionId, buildingId, state) {
-  return state.buildings[buildingId].answers[questionId]
+    return state.buildings[buildingId].answers[questionId]
 }
+
+//returns (from the questions provided) how many are remaining to answer
```

Comment:
Thanks for the inline documentation! Can you specify what happens when `buildingId` or `questions` are not provided?